.github/actions/deploy/action.yml

```diff
@@ -53,6 +53,9 @@ inputs:
   doppler_token:
     required: false
     description: 'Doppler token for accessing environment variables'
+  workflow_run_number:
```

Comment:
We should probably change this to `deploy_id`.
Workflow number is just a means of providing an ID to the deployment.

Reply:
Yeah, sure